README.md

```diff
@@ -1,110 +1,38 @@
+### Reference files:
 
-## Introduction
+`test/reference/`
 
-**nf-core/variantcall** is a bioinformatics pipeline that ...
+`hg19.fa` *Not provided, download needed* https://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/hg19.fa.gz
 
-<!-- TODO nf-core:
-   Complete this sentence with a 2-3 sentence summary of what types of data the pipeline ingests, a brief overview of the
-   major pipeline sections and the types of output it produces. You're giving an overview to someone new
-   to nf-core here, in 15-20 seconds. For an example, see https://github.com/nf-core/rnaseq/blob/master/README.md#introduction
--->
+`hg19.fa.fai`
 
-<!-- TODO nf-core: Include a figure that guides the user through the major workflow steps. Many nf-core
-     workflows use the "tube map" design for that. See https://nf-co.re/docs/contributing/design_guidelines#examples for examples.   -->
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
+`hg38.fa.gz` *Not provided, download needed* https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz
 
-1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+`hg38.fa.fai`
 
-## Usage
+`hg38.dict`
 
-> [!NOTE]
-> If you are new to Nextflow and nf-core, please refer to [this page](https://nf-co.re/docs/usage/installation) on how to set-up Nextflow. Make sure to [test your setup](https://nf-co.re/docs/usage/introduction#how-to-run-a-pipeline) with `-profile test` before running the workflow on actual data.
+`hg19ToHg38.over.chain.gz` https://hgdownload.soe.ucsc.edu/goldenPath/hg19/liftOver/hg19ToHg38.over.chain.gz
 
-<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
-     Explain what rows and columns represent. For instance (please edit as appropriate):
-
-First, prepare a samplesheet with your input data that looks as follows:
-
-`samplesheet.csv`:
-
-```csv
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-```
-
-Each row represents a fastq file (single-end) or a pair of fastq files (paired end).
-
--->
-
-Now, you can run the pipeline using:
-
-<!-- TODO nf-core: update the following command to include all required parameters for a minimal example -->
+### Usage
 
+1. Download `hg19.fa` and `hg38.fa.gz` to `test/reference/`
+2. Retrive `api_token`
+3. Workflow command
 ```bash
-nextflow run nf-core/variantcall \
-   -profile <docker/singularity/.../institute> \
-   --input samplesheet.csv \
-   --outdir <OUTDIR>
+nextflow run main.nf \
+    -profile docker,test,rdpc_qa \
+    --api_token NNNNNNN \
+    --hg19_ref_fa /PATH/TO/hg19.fa \
+    --hg19_ref_fai /PATH/TO/hg19.fa.fai \
+    --hg19_to_hg38_chain /PATH/TO/hg19ToHg38.over.chain.gz \
+    --hg38_ref_fa /PATH/TO/hg38.fa.gz \
+    --hg38_ref_fai /PATH/TO/hg38.fa.fai \
+    --hg38_ref_dict /PATH/TO/hg38.dict
```

Comment:
full command provided

Reply:
add another example where test profile is not called and xml and experiment info are provided.